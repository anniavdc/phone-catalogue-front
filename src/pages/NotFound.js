import React from 'react';
import { Link } from 'react-router-dom';

// files
import searchIcon from './../assets/search-icon.png'

export default function NotFound() {
    return (
        <div className="not-found-wrapper">
            <div className="not-found-icon-wrapper">
                <img src={searchIcon} alt="not found icon" className="search-icon" />
                <p className="code">404</p>
            </div>
            <div className="not-found-content">
                <p>P치gina no encontrada</p>
                <p>Lo sentimos. No hemos podido encontrar la p치gina que est치s buscando</p>
            </div>
            <Link className="link" to="/app">
                Ir a p치gina principal
            </Link>
        </div>
    )
}
