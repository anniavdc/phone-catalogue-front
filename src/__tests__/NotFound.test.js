import React from 'react';

// files
import render from './../utils/tests/renderWithRouter';
import NotFound from '../pages/NotFound';

describe('NotFound', () => {
    test('render', () => {
        const { container } = render(
            <NotFound />
        );
        expect(container).toMatchSnapshot();
    });

})
