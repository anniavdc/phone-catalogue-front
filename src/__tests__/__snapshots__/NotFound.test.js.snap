// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound render 1`] = `
<div>
  <div
    class="not-found-wrapper"
  >
    <div
      class="not-found-icon-wrapper"
    >
      <img
        alt="not found icon"
        class="search-icon"
        src="search-icon.png"
      />
      <p
        class="code"
      >
        404
      </p>
    </div>
    <div
      class="not-found-content"
    >
      <p>
        Page not found
      </p>
      <p>
        Sorry. We couldn't find the page you are looking for
      </p>
    </div>
    <a
      class="link"
      href="/app"
    >
      Go to main page
    </a>
  </div>
</div>
`;
